.github/workflows/linters.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   rubocop:
     name: Rubocop
     runs-on: ubuntu-22.04
-
+    
```

Comment:
![image](https://github.com/AbuTalha3/tdd-ruby-spec/assets/93039359/c4e6fbb6-26b0-4e37-b006-4aa2d64637a2)
![image](https://github.com/AbuTalha3/tdd-ruby-spec/assets/93039359/37acd531-3852-4f9f-8f55-e518e3121f37)

- [ ] It would be nice if you fix the highlighted link in the images above 👀 to make your readme descriptive and professional 👍